Task_1_Firsov_Aleksei

grifus3000 opened the pull request:
Task_1

grifus3000 committed:
stable version

grifus3000 committed:
stable with scroll view

grifus3000 committed:
final

ios-school-admin commented on SplitVC/SplitVC/DetailVC.swift:
```diff
@@ -0,0 +1,67 @@
+
+import UIKit
+
+class DetailVC: UIViewController {
```
Не стоит так называть ViewController-ы, DetailViewController смотрелось бы лучше

ios-school-admin commented on SplitVC/SplitVC/DetailVC.swift:
```diff
@@ -0,0 +1,67 @@
+        
+        for _ in 1...5 - ratingInt {
+            ratingStars += "☆"
+        }
```
Гениально и просто :D

ios-school-admin commented on SplitVC/SplitVC/TableVC.swift:
```diff
@@ -0,0 +1,121 @@
+
+class TableVC: UITableViewController {
+    
+    public static let dataArr: [Item] = [
```
Массив для TableView не должен быть static. Для передачи данных между вьюконтроллерами лучше обращаться через общий splitViewController, либо использовать Data Binding

ios-school-admin commented on SplitVC/SplitVC/TableVC.swift:
```diff
@@ -0,0 +1,121 @@
+            cell.priceLable.text = ""
+        }
+        cell.cellImage.image = UIImage(named: TableVC.dataArr[indexPath.row].imageName)
+
```
Чтобы ViewController не разрастался, в методе cellForRowAt лучше передавать объект внутрь cell написав и вызвав 
```
update(by item: Item) {
   titleLabel.text = item.title
  // и т.д.
}
```

ios-school-admin commented on SplitVC/SplitVC/MovieTableViewCell.swift:
```diff
@@ -0,0 +1,35 @@
+
+        // Configure the view for the selected state
+    }
+
```
Где-нибудь здесь лучше сделать метод update(by: Item), где будет происходить апдейт ячейки в соответствии с Item

grifus3000 committed:
changed controllers name
using delegate for data binding
modified transmit data to cell and detail view
fixing some things
